### ISSUE TITLE
[OpenBMC] - Remove Transient State Processing until code is stabilized.

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3808]
With the OpenBMC firmware, talking to the development team, they mention that we should NOT use the RequestedPowerTransition attribute , but could use the RequestedHostTransition Attribute to detect some transient states.  

For example, `powering-on` and `powering-off`   

But right now the firmware is a little unstable, so I think it's best that we remove this transient state and ONLY report back On or Off.  

This will cause some incorrect state reporting, for example, power off takes a few minutes, xCAT will show:
```
rpower cn01 off
cn01: off
```

Then while it powers off.... 
```
rpower cn01 state
cn01: on
````

Then finally will go to off when the state changes.  But this is better than what we are seeing today...

```
[root@briggs01 ~]# rpower node-70e28414280d state
node-70e28414280d: [openbmc_debug] POST https: //172.12.253.119/login -d { "data": [ "root", "0penBmc" ] }
node-70e28414280d: [openbmc_debug] login_response 200 OK
node-70e28414280d: [openbmc_debug] GET https: //172.12.253.119/xyz/openbmc_project/state/enumerate
node-70e28414280d: [openbmc_debug] rpower_status_response 200 OK
node-70e28414280d: [openbmc_debug] State CurrentBMCState=xyz.openbmc_project.State.BMC.BMCState.Ready
node-70e28414280d: [openbmc_debug] State RequestedBMCTransition=xyz.openbmc_project.State.BMC.Transition.None
node-70e28414280d: [openbmc_debug] State CurrentPowerState=xyz.openbmc_project.State.Chassis.PowerState.On
node-70e28414280d: [openbmc_debug] State RequestedPowerTransition=xyz.openbmc_project.State.Chassis.Transition.Off
node-70e28414280d: [openbmc_debug] State CurrentHostState=xyz.openbmc_project.State.Host.HostState.Running
node-70e28414280d: [openbmc_debug] State RequestedHostTransition=xyz.openbmc_project.State.Host.Transition.Off
node-70e28414280d: powering-off
```

This node is `powering-off` but I don't know who sent that transition request.  The compute side is running and no indication of a power off has been sent.

**Comment by whowutwut** [comment]
@xuweibj Could you make this high priority to complete asap, We want the states to report correctly until the firmware is more stable to detect transition states.  I'm OK leaving the code logic in there for now so that we could test it more easily when we get more stable firmware.

**Comment by xuweibj** [comment]
@whowutwut this issue is fixed in pull request #3812, shall we close this issue?
